applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -238,6 +252,25 @@ def InitializeSolutionStep(self):
             # Initialize the solver current step
             self._GetSolutionStrategy().InitializeSolutionStep()
 
+    def SolveSolutionStep(self):
+        super(NavierStokesTwoFluidsSolver, self).SolveSolutionStep()
```

Comment:
```suggestion
        super().SolveSolutionStep()
```

not needed any more since we dropped python2 support